scripts/insert_into_appointments.sql

```diff
@@ -1 +1,29 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments('date', 'time', patient_id, doctor_id)
```

Comment:
This will cause syntax errors similar to the problem of trying to create `'date'` and `'time'` as column names.